instructure/common-cartridge-viewer
# puts ModulesList inside a <ul>

test plan:
- view a modules index in the viewer
- run axe chrome extension
- it no longer reports
- li not in a ul violation
